include/hw/i386/virt.h

```diff
@@ -76,4 +76,11 @@ MemoryRegion *virt_memory_init(VirtMachineState *vms);
 
 DeviceState *virt_acpi_init(qemu_irq *gsi, PCIBus **pci_bus, uint16_t segment_total);
 
+#define PCI_HOST_PCIEXBAR_BASE 0x40000000
```

Comment:
@liujing2 ideally I want to pass this via fw_cfg so that the firware does not hardcode this value.

Reply:
OK, let me do that.